src/app/profile/page.tsx

```diff
@@ -10,7 +10,9 @@ const ProfilePage = async () => {
       <h1 className="my-4 mb-8 text-center text-3xl font-bold text-gray-900 dark:text-white">
         Edit Profile
       </h1>
-      {isBeforeFirstMatch() && <ProfileForm profile={profile} />}
+      {isBeforeFirstMatch() && (
+        <ProfileForm profile={JSON.parse(JSON.stringify(profile))} />
```

Comment:
it would be great not to use JSON... you probably have some ID as object not string, or some additional id in response